### PR TITLE
Archive Azure 15.1.0

### PR DESCRIPTION
**Pull request by T-Kukawka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ to all Giant Swarm installations.
     - [v15.0.0-beta2](https://github.com/giantswarm/releases/tree/master/azure/archived/v15.0.0-beta2)
     - [v15.0.0-beta1](https://github.com/giantswarm/releases/tree/master/azure/archived/v15.0.0-beta1)
   - v15.1
-    - [v15.1.0](https://github.com/giantswarm/releases/tree/master/azure/v15.1.0)
+    - [v15.1.0](https://github.com/giantswarm/releases/tree/master/azure/archived/v15.1.0)
     - [v15.1.1](https://github.com/giantswarm/releases/tree/master/azure/v15.1.1)
```
Comment (whites11) [inline README.md:152]: Shouldn't 15.1.1 be above 15.1.0?

Comment (whites11) [reply README.md:152]: Yes it should, please swap the two lines

Comment (T-Kukawka) [reply README.md:152]: i guess somebody did that with release, can change it

Comment (T-Kukawka) [reply README.md:151]: fixed

**Review by whites11 — APPROVED** [review]

**Pull request merged**